src/SFA.DAS.EAS.Web/Views/Transfers/TransferDetails.cshtml

```diff
@@ -100,16 +91,15 @@
                 </div>
             </fieldset>
             <button type="submit" class="button" id="submit-transfer-connection">Continue</button>
-            <a class="button-link" href="@Url.Action("Index", "EmployerCommitments")" aria-label="Cancel">Cancel</a>
             @Html.AntiForgeryToken()
-            @Html.Hidden("HashedTransferReceiverAccountId", Model.Data.HashedTransferReceiverAccountId)
+            @Html.Hidden("TransferReceiverName", Model.Data.LegalEntityName)
         </form>
     </div>
 </div>
 @section breadcrumb {
-    <div class="breadcrumbs">
-        <a href="@Url.Action("Index", "EmployerCommitments")" aria-label="Back to Apprentices" class="back-link">Back to Apprentices</a>
-    </div>
+<div class="breadcrumbs">
+    <a href="@Url.ExternalUrlAction("teams", "transfers")">Back to Transfers</a>
```

Comment:
Have you confirmed this as the correct controller action with MYA team? "Teams" doesn't sound right to me. The screenshot of the prototype only says "Back", so is the label correct?